### PR TITLE
MOSIP-29490, MOSIP-29848, MOSIP-22042 : Updated the SBI Filters & Updated the error messages

### PR DESCRIPTION
**Pull request by GOKULRAJ136**

### CODE REVIEW

**Review by balaji-alluru — APPROVED** [review]

File: artifacts/pom.xml [hunk 1]
```diff
@@ -99,7 +99,7 @@
 		<!-- mock-sdk -->
 		<mock-sdk.version>1.2.0.1-SNAPSHOT</mock-sdk.version>
 		<mock-sdk.location>/usr/share/nginx/html/artifactory/libs-release-local/mock-sdk/1.1.5</mock-sdk.location>
-		<mock-sdk.fileName>mock-sdk-jar-with-dependencies.jar</mock-sdk.fileName>
```
Comment (ase-101) [inline artifacts/pom.xml:102]: why is this line removed?

**Comment by ase-101** [comment]
Invalid PR
